settings.gradle

```
@@ -1,8 +1,16 @@
 pluginManagement {
     plugins {
         id 'org.jetbrains.kotlin.android' version '1.4.31'
+        id "com.automattic.android.publish-to-s3" version "0.7.0"
```

Annoying nitpick. The line above uses single quotes 😅 
```suggestion
        id 'com.automattic.android.publish-to-s3' version '0.7.0'
```